package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "vui-table",
+  "version": "1.0.0",
```

Comment:
This is brand new repo, so I would start it at 0.0.1.  Once we are confident in the styles and the dust settles, we can bump the version to 1.0.0 if we want to.  That being said, while many (perhaps all?) of the existing VUI components were bumped to version 1.x for C12, and that is convenient for our ongoing development, it's possible they all won't be instep like that.